app/src/lib/stores/app-store.ts

```diff
@@ -5190,6 +5200,14 @@ export class AppStore extends TypedBaseStore<IAppState> {
     return Promise.resolve()
   }
 
+  public _setConfirmUndoCommitSetting(value: boolean): Promise<void> {
+    this.confirmUndoCommit = value
+    setBoolean(confirmForcePushKey, value)
+
+    this.emitUpdate()
+
+    return Promise.resolve()
+  }
```

Comment:
Nit: add blank line after this:
```suggestion
  }

```

Reply:
done